django_usaid_clinicfinder/clinicfinder/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models as djangomodels
+from django.contrib.gis.db import models as gismodels
+from django_hstore import hstore
+
+
+class HStoreModel(djangomodels.Model):
+    objects = hstore.HStoreManager()
+
+    class Meta:
+        abstract = True
+
+
+class Location(gismodels.Model):
+    point = gismodels.PointField()
+    created_at = djangomodels.DateTimeField(auto_now_add=True)
+    updated_at = djangomodels.DateTimeField(auto_now=True)
+
+    # GeoDjango-specific overriding the default manager with a
+    # GeoManager instance.
+    objects = gismodels.GeoManager()
+
+    def __unicode__(self):
+        return "%s" % (self.point)
+
+
+class PointOfInterest(HStoreModel):
```

Comment:
It's designed like this with two things in mind:

1. Reuse: locations being geo points and point of interest being "stuff at a location". Think about Praekelt's building for example. It is one location but there is many points of interest. When you move, it'll be a change of location. Not "change the Location" (edit location) but "change of Location" (create location or attach to location). 
2. Limitations of using two extensions. We're using hstore for its ability to have a dynamic model for the data on a point of interest, and gis for the Point support.